Lombiq.Hosting.Tenants.Maintenance/Models/StaggeredMaintenanceOptions.cs

```diff
@@ -0,0 +1,8 @@
+namespace Lombiq.Hosting.Tenants.Maintenance.Models;
+
+public class StaggeredMaintenanceOptions
+{
+    public int? MillisecondBetweenBatches { get; set; }
```

Comment:
This can be just seconds for simplicity.
```suggestion
    public int? BatchIntervalSeconds { get; set; }
```